spec/features/courses_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+include CourseHelper
```

Comment:
The fact that this functionality is included in a helper is something I don't love. It's then globally available. Your argument for that might be that we need it in the jbuilder as well as in the views, but that could be handled by a presenter or service object. Exposing that information to the global namespace is, to me, not preferable to wrapping this functionality in a separate object.